layer.go

```diff
@@ -68,9 +67,30 @@ func (p Profile) ProcessAddf(processType string, name string, format string, a .
 	p.Addf(filepath.Join(processType, name), format, a...)
 }
 
-// Contribute represents a layer managed by the buildpack.
-type Layer struct {
+// BOMFormat indicates the format of the SBOM entry
+type BOMFormat int
+
+const (
+	CycloneDXJSON BOMFormat = iota
+	SPDXJSON
+	SyftJSON
+)
+
+func (b BOMFormat) String() string {
+	return []string{"cdx.json", "spdx.json", "syft.json"}[b]
+}
+
+// BOMLocation indicates the location of the SBOM entry
+type BOMLocation int
+
+const (
+	BuildBOM BOMLocation = iota
+	LaunchBOM
+	LayerBOM
+)
```

Comment:
We shouldn't need this anymore.

Reply:
Weird. I thought I had removed this. Definitely removed now.